Dockerfile.rhel7

```diff
@@ -1,8 +1,4 @@
-<<<<<<< HEAD
```

Comment:
This shouldn't exist in a sync PR.

Reply:
I believe that is outdated that line no longer exists.

Reply:
I don't. This is outdated, this is not my own PR. I've skipped this https://github.com/openshift/ib-sriov-cni/commit/b9e8a4a94756df17b70c78b577d7e1a5cd14ac8c. So that is why the dockerfile is there. I just skipped that PR.

Reply:
Look at https://github.com/openshift/ib-sriov-cni/pull/33/commits and scroll all the way down.

Reply:
aaah I see. whoops I must have not removed that commit properly. Will do that.